.travis.yml

```diff
@@ -2,12 +2,6 @@ language: d
 
 d:
   - dmd-2.067.1
-  - dmd-2.066.1
-  - dmd-2.065.0
-  - ldc-0.14.0
-  - ldc-0.15.1
-  - gdc-4.9.0
-  - gdc-4.9.2
```

Comment:
Yeah, travis.yml and dub.json changes are still temporary, because Botan doesn't compile yet on most platforms. This will need some special care before merging.